src/Core/LoginConfig/LoginConfig.php

```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\LoginConfig;
+
+use Shopware\Core\Framework\Bundle;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\LoginConfig\DependencyInjection\LoginConfigCompilerPass;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
+
+/**
+ * @internal
+ *
+ * @codeCoverageIgnore
+ */
+#[Package('core')]
+class LoginConfig extends Bundle
```

Comment:
Ah maybe you made the service nullable because it comes from a different bundle that could be deactivated? We should stay inside the core / admin bundle